content/changes.md

```diff
@@ -8,6 +8,46 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
+## January, 2024 
```

Comment:
I would pick a specific date here so it's in keeping with the rest of the product updates, or use "January 2024" without a comma if you do want to switch to a full month. For the backdated entries, you could use an approximate date or January 1, 2024. I think having the exact date is nice, personally, so there's a clear date of release for the features.

Reply:
I agree but since we are going to go to a monthly cadence I prefer to keep the monthly format and at the end of each month or at the beginning of the month about the closed month. I understand that we want to keep the format but in any case I would almost go for the **end of the month** instead of the beginning. Now it is a bit confusing because I am generating months already past. It could be January 31, 2024.

Reply:
If you use the same day every month, that's fine with me. January 31, 2024 and so on works for me. I think this is better than "January 2024" which is a bit vague. Once we catch up, we'll want to use the exact day it's posted as the date.

Reply:
That is why I don't want to be attached to an specific date, I prefer at this point to go to a more monthly cadence without matter the exact date just the updates or improvements released during the last month.